safe-global/safe-wallet-web
# fix: Bug fixes and improvements for Dark Mode

## What it solves

Resolves #3066 

## How this PR fixes it
This pull request addresses color inconsistencies related to the color palette scheme and dark mode styles. The following changes have been made:

* [x] secondary:background is `#1B2A22`. previously(`#303033`) - no effect on custom component styles.
* [x] The selected network color in the network dropdown is changed to `#303033`. previously(`#1B2A22`).
* [x] In the Safe apps component styles_content__TMlnS now uses `#12FF80` for the border and the arrow component
     - __can we change the hex code for secondary:main in darkPalette.ts to `#12FF80`?__ 
     if we change this colour definition, custom theme definitions could be overidded causing color inconsistencies for few custom component, to avoid this, explicit color definitions were directly added to this component's css (https://github.com/manoranjith-shankar/safe-wallet-web/commit/1756419ba7f6ac682646e3d2aa5022d7778f5927).

### changes made (https://github.com/safe-global/safe-wallet-web/pull/3102/commits/0babf13e3bda91f2f764e99445440cab0ec3b403)

* [x] success:background is set to `#1f2920` by fixing the typo for alerts in `safeTheme.ts`.
* [x] Background of the info notification color is now `#19252C` in dark mode.
* [x] In the Safe accounts list, the color of the addresses is set to be text:primary.

## How to test it

* Go to Safe Apps, see the sliding component above;
* Open the network switcher in the top navbar.
* Go to Settings > Notifications and Settings > Modules to see alert banners;
* Go to any page, open the Safe Accounts sidebar;
* Open a Safe Accounts with pending notifications to trigger Info alert;

## Screenshots
![PR-2-1](https://github.com/safe-global/safe-wallet-web/assets/92922580/264c6777-bf1a-4acd-9891-4597a769a25e)
![PR-2-2](https://github.com/safe-global/safe-wallet-web/assets/92922580/a63ed6ca-9aa6-423e-a7d9-3097031d1ec6)
![PR-2-3](https://github.com/safe-global/safe-wallet-web/assets/92922580/163148f1-b6e9-4b38-b5bf-d7c79662d8cd)
![PR-2-4](https://github.com/safe-global/safe-wallet-web/assets/92922580/76c35261-11a1-4785-aabc-62cf7dbb516a)
![PR-2-5](https://github.com/safe-global/safe-wallet-web/assets/92922580/0f4b3c00-bfe3-407c-a97e-d9641e78dba2)
![PR-2-6](https://github.com/safe-global/safe-wallet-web/assets/92922580/8f38b8a7-1913-4ed6-b96d-d04024406283)

## Checklist
* [x] I've tested on light theme for any color inconsistencies
